### PR TITLE
Add support for private poi files in tyr

### PR DESCRIPTION
**Pull request by crocme10**
* Adds a new binarisation step for poi2mimir
* Calls that binarisation for family_type 'poi'

### CODE REVIEW

File: source/tyr/tyr/default_settings.py [hunk 1]
```diff
@@ -14,13 +14,13 @@
 # URI for postgresql
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 # http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-SQLALCHEMY_DATABASE_URI = 'postgresql://navitia:navitia@localhost/jormungandr'
+SQLALCHEMY_DATABASE_URI = 'postgresql://navitia:navitia@10.0.3.14/jormungandr'
```
Comment (benoit-bst) [inline source/tyr/tyr/default_settings.py]: Oups, :speak_no_evil: the `default_settings.py` has to be the same

File: source/tyr/tyr/default_settings.py [hunk 2]
```diff
@@ -14,13 +14,13 @@
...
 
 
 # URI for cities database
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 # http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-CITIES_DATABASE_URI = os.getenv('TYR_CITIES_DATABASE_URI', 'postgresql://navitia:navitia@localhost/cities')
+CITIES_DATABASE_URI = os.getenv('TYR_CITIES_DATABASE_URI', 'postgresql://navitia:navitia@10.0.3.14/cities')
```
Comment (benoit-bst) [inline source/tyr/tyr/default_settings.py]: Oups, :speak_no_evil: the `default_settings.py` has to be the same

**Review by kinnou02 — COMMENTED** [review]
Black isn't happy :(

File: source/tyr/tyr/default_settings.py (hunk 2 above)
Comment (crocme10) [reply source/tyr/tyr/default_settings.py]: :+1:

**Pull request closed without merging**